test/parallel/test-repl-.editor.js

```diff
@@ -31,6 +31,17 @@ function run({input, output, event}) {
   stream.emit('data', input);
   replServer.write('', event);
   replServer.close();
+
+  if (!checkTerminalCodes) {
+    while (found.includes(terminalCode))
+      found = found.replace(terminalCode, '');
```

Comment:
@addaleax 👍 Just to convey the term 'quoted' correctly, added code snippet with `[` escaped manually 😄